src/components/App/index.js

```diff
@@ -16,6 +17,7 @@ export default class App extends React.Component {
         <HeaderContainer genre={genre} pathname={pathname} />
           {children}
         <PlaylistContainer />
+        <VolumeContainer />
```

Comment:
At this point one could argue to put both container components, VolumeContainer and PlaylistContainer, into the PlayerComponent. Otherwise with a growing Player functionality the list of components around the Player will grow. But this is a refactoring for the future.